scripts/make_docs.py

```diff
@@ -57,16 +59,23 @@
 
 ## Available selectors
 
-{%- for key, val in platforms|dictsort %}
-- `{{key}}`
-{% endfor %}
```

Comment:
This seemed like a mistake. LMK if I should add it back

Reply:
Yes, I meant the extra lines between each bullet. It looked a little funny to me in the rendered page.
Thanks, I'll use that